sample/kotlin/src/main/kotlin/com/datadog/android/sample/logs/LogsFragment.kt

```diff
@@ -87,12 +105,23 @@ class LogsFragment :
 
     // region NDK
 
-    external fun simulateNdkCrash()
+    external fun simulateNdkCrash(signal: Int)
 
     // endregion
 
+    private data class NdkCrashType(val signal: Int, val label: String) {
+        override fun toString(): String {
+            return label
+        }
+    }
+
     companion object {
 
+        // NDK Crash signals
+        const val SIGILL = 4
+        const val SIGABRT = 6
+        const val SIGSEGV = 11
```

Comment:
As I understand, those values (`4`, `6`, `11`) correspond to cpp. Maybe we can express it more, at least with a comment? Now they read as a bunch of random values, hard to grasp outside of this PR context.